test/functional/tests/cli/test_seq_cutoff.py

```diff
@@ -0,0 +1,290 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+import pytest
+import random
+from ctypes import c_uint32
+from api.cas import casadm
+from api.cas.cache_config import SeqCutOffPolicy
+from api.cas.core import SEQ_CUTOFF_THRESHOLD_MAX, SEQ_CUT_OFF_THRESHOLD_DEFAULT
+from api.cas.casadm import set_param_cutoff_cmd
+from core.test_run import TestRun
+
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_seq_cutoff_default_params():
+    """
+    title: Default sequential cut-off threshold & policy test
+    description: Test if proper default threshold and policy is set after cache start
+    pass_criteria:
+        - "Full" shall be default sequential cut-off policy
+        - There shall be default 1MiB (1024kiB) value for sequential cut-off threshold
+    """
+    with TestRun.LOGGER.step("Test prepare (start cache and add core)"):
+        cache, cores = prepare()
+
+    with TestRun.LOGGER.step("Getting sequential cut-off parameters"):
+        params = cores[0].get_seq_cut_off_parameters()
+
+    with TestRun.LOGGER.step("Check if proper sequential cut off policy is set as a default"):
+        if params.policy != SeqCutOffPolicy.DEFAULT:
+            TestRun.LOGGER.error(f"Wrong sequential cut off policy set: {params.policy} "
+                                 f"should be {SeqCutOffPolicy.DEFAULT}")
+
+    with TestRun.LOGGER.step("Check if proper sequential cut off threshold is set as a default"):
+        if params.threshold != SEQ_CUT_OFF_THRESHOLD_DEFAULT:
+            TestRun.LOGGER.error(f"Wrong sequential cut off threshold set: {params.threshold} "
+                                 f"should be {SEQ_CUT_OFF_THRESHOLD_DEFAULT}")
+
+
+@pytest.mark.parametrize("policy", SeqCutOffPolicy)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_seq_cutoff_set_get_policy_core(policy):
+    """
+    title: Sequential cut-off policy set/get test for core
+    description: |
+        Test if CAS is setting proper sequential cut-off policy for core and
+        returns previously set value
+    pass_criteria: |
+        - Sequential cut-off policy obtained from get-param command for the first core must be
+        the same as the one used in set-param command
```

Comment:
BTW this yaml is overindented. It's recommended to use 2 spaces (see example in GUIDELINES,md). Using 4 spaces doesn't make yaml incorrect, but let's keep it consistent.